tsl/test/sql/cagg_joins.sql.in

```diff
@@ -153,6 +162,51 @@ SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
 FROM conditions JOIN devices USING (device_id)
 GROUP BY name, bucket;
 
+SELECT * FROM conditions_summary_daily_3_reorder ORDER BY bucket;
+
+CREATE MATERIALIZED VIEW conditions_summary_daily_3_reorder_using_realtime
+WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions JOIN devices USING (device_id)
+GROUP BY name, bucket;
+
+SELECT * FROM conditions_summary_daily_3_reorder_using_realtime ORDER BY bucket;
+
+--Create CAgg with join and additional WHERE conditions
+CREATE MATERIALIZED VIEW conditions_summary_daily_more_conds
+WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions JOIN devices ON conditions.device_id = devices.device_id
```

Comment:
I don't understand this comment, this is a test for caggs with joins with more conditions. What we support is additional condition in where clause. Did you mean to add this also in repair test?

Reply:
Yes, I was thinking about joins with multiple `ON` conditions, for example, `FROM conditions JOIN devices ON conditions.device_id = devices.device_id AND conditions.building_id = devices.building_id`. Since this is a quite drastic change and the checks are not easy to get right, it would be good to ensure that we haven't missed the obvious cases.

Reply:
@mkindahl currently we support only 1 (one) join condition... for explicit INNER JOIN extra conditions should be placed into the WHERE clause, but not for JOIN...

@RafiaSabih would be good to add a failing test like `FROM conditions JOIN devices ON conditions.device_id = devices.device_id WHERE conditions.building_id = devices.building_id`

Reply:
Multiple ON conditions is not supported.